.github/ISSUE_TEMPLATE/crash_report.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Crash report
+about: Submit a crash report generated by redis-server
+title: '[CRASH] REDIS BUG REPORT'
+labels: ''
+assignees: ''
+
+---
+
+**Crash report**
+
+Paste the relevant log lines below.
```

Comment:
```suggestion
Paste the complete crash log between the quotes below.
please include a few lines from before the crash log too in case they provide context.
```